kit-sdq/programming-lecture-eclipse-artemis
# Adopt: Cookie Login & Version Management

### Checklist
- [x] I tested *all* changes and their related features locally or with a test instance of Artemis.

### Motivation and Context


See https://github.com/ls1intum/Artemis/pull/5723

### Description

This PR changes the authentication from authorization headers (bearer tokens) to cookies as introduced in https://github.com/ls1intum/Artemis/pull/5723

### Steps for Testing

1. Create a local instance of Artemis version https://github.com/ls1intum/Artemis/pull/5723
2. Verify the in browser login process for student edition
3. Submit a change to a programming exercise to check websockets
